filesystem/btrfs_stream.ksy

```diff
@@ -0,0 +1,151 @@
+meta:
+  id: btrfs_stream
+  application: Btrfs
+  file-extension: btrfs-stream
+  endian: le
+  license: CC0-1.0
+doc: |
+  This spec can be used to disassemble raw Btrfs Send stream format
+  version 1 from the output of `btrfs send` command.
+  See also the `btrfs receive --dump` command.
```

Comment:
A minor suggestion: it would be beneficial to include brief information on:

* What btrfs itself is?
* What this stream functionality is all about?
* How stream functionality and commands illustrated are related?

Reply:
Brief introduction added